inc/targetticket.class.php

```diff
@@ -1112,8 +1110,9 @@ public function save(PluginFormcreatorForm_Answer $formanswer) {
       switch ($this->fields['destination_entity']) {
          // Requester's entity
          case 'current' :
-            $data['entities_id'] = $_SESSION['glpiactive_entity'];
+            $data['entities_id'] = $formanswer->getField('entities_id');
```

Comment:
I 'm not ok with this modification.
'current' case, if for current active entity
form entity is 'form' case

Reply:
After checking in the UI : 
- the comment **Requester's entity** actually matches **case 'current'** (I  checked, just in case).
- in the UI, this settings is displayed as **active entity**. This is not clear if this is the active entity ofthe requester or the validator. In my opinion, this is  the requester's active entity. Maybe we should update  the label matching this case.
- when a requester saves a form, his current entity is saved in  the form answers. This change retrieves this entity as it reflects the active entity of the requester